config/coreboot-qemu.config

```diff
@@ -1,18 +1,20 @@
+CONFIG_LOCALVERSION="Heads-v0.2.0-947-g979fad3-dirty"
```

Comment:
Should not be specified anymore. Is injected automatically since https://github.com/osresearch/heads/commit/bd7a945bbbf586ceb7c83785ef36d95f0adf7a15

Reply:
fixed in #895. Waiting for build success prior of merging. Then just rebase please @osresearch